.circleci/config.yml

```diff
@@ -332,25 +419,45 @@ jobs:
   yaml-linter:
     executor: node
     steps:
-      - run-yarn-command:
-          command-name: Lint YAML files
-          command: lint-yaml `find resources -name '*.yaml'`
+      - attach-workspace
+      - create-checksum-file:
+          filename: .YAML-CHECKSUMS
+          find-args: ". -type f -name '*.yaml' -or -name '*.yml'"
+      - run-on-change:
+          # package.json is included in the cache key in case the yamllint dependency changes
+          cache-key: lint-yaml-{{ checksum ".YAML-CHECKSUMS" }}-{{ checksum "package.json" }}
+          steps:
+            - run-yarn-command:
+                command-name: Lint YAML files
+                command: lint-yaml `find resources -name '*.yaml'`
```

Comment:
this needs `*.yml` too to match the `find-args` above

Reply:
Didn't @nemanjaglumac tweak `package.json` so that you need pass any arguments to (yarn) lint-yaml anymore?